lib/elixir/lib/option_parser.ex

```diff
@@ -499,4 +562,28 @@ defmodule OptionParser do
   defp negative_number?(arg) do
     match?({_, ""}, Float.parse(arg))
   end
+
+  defp format_errors(errors, opts) do
+    details = Enum.map_join(errors, ". ", &format_error(&1, opts))
+    total = Enum.count(errors)
+    error = if total == 1, do: "error", else: "errors"
+    "#{total} #{error} found! #{details}"
```

Comment:
@josevalim ah, awesome then. If you're going to merge this PR in the current state, give a look at my comments then :)